File: python/Monitor.py
class Monitor:
    def __init__(self, merk="", model="", ukuran_inch=0.0, resolusi_x=0, resolusi_y=0, refresh_rate=0):
        self.merk = merk
        self.model = model
        self.ukuran_inch = ukuran_inch
        self.resolusi_x = resolusi_x
        self.resolusi_y = resolusi_y
        self.refresh_rate = refresh_rate
    
    def set_merk(self, merk):
        self.merk = merk
    
    def set_model(self, model):
        self.model = model
    
    def set_ukuran_inch(self, ukuran_inch):
        self.ukuran_inch = ukuran_inch
    
    def set_resolusi(self, resolusi_x, resolusi_y):
        self.resolusi_x = resolusi_x
        self.resolusi_y = resolusi_y
    
    def set_refresh_rate(self, refresh_rate):
        self.refresh_rate = refresh_rate
    
    def get_merk(self):
        return self.merk
    
    def get_model(self):
        return self.model
    
    def get_ukuran_inch(self):
        return self.ukuran_inch
    
    def get_resolusi_x(self):
        return self.resolusi_x
    
    def get_resolusi_y(self):
        return self.resolusi_y
    
    def get_refresh_rate(self):
        return self.refresh_rate
    
    def display_info(self):
        print(f"{self.merk} {self.model} {self.ukuran_inch}\" {self.resolusi_x}x{self.resolusi_y} {self.refresh_rate}Hz", end="")

File: cpp/Monitor.cpp
#ifndef MONITOR_CPP
#define MONITOR_CPP

#include <iostream>
#include <string>

using namespace std;

class Monitor {
private:
    string merk;
    string model;
    float ukuranInch;
    int resolusiX;
    int resolusiY;
    int refreshRate;
public:
    Monitor() {}

    Monitor(string merk, string model, float ukuranInch, int resolusiX, int resolusiY, int refreshRate) {
        this->merk = merk;
        this->model = model;
        this->ukuranInch = ukuranInch;
        this->resolusiX = resolusiX;
        this->resolusiY = resolusiY;
        this->refreshRate = refreshRate;
    }

    void setMerk(string merk) {
        this->merk = merk;
    }

    void setModel(string model) {
        this->model = model;
    }

    void setUkuranInch(float ukuranInch) {
        this->ukuranInch = ukuranInch;
    }

    void setResolusi(int resolusiX, int resolusiY) {
        this->resolusiX = resolusiX;
        this->resolusiY = resolusiY;
    }

    void setRefreshRate(int refreshRate) {
        this->refreshRate = refreshRate;
    }

    string getMerk() {
        return this->merk;
    }

    string getModel() {
        return this->model;
    }

    float getUkuranInch() {
        return this->ukuranInch;
    }

    int getResolusiX() {
        return this->resolusiX;
    }

    int getResolusiY() {
        return this->resolusiY;
    }

    int getRefreshRate() {
        return this->refreshRate;
    }

    void displayInfo() {
        cout << merk << " " << model << " " << ukuranInch << "\" " 
             << resolusiX << "x" << resolusiY << " " << refreshRate << "Hz";
    }

    ~Monitor() {}
};

#endif // MONITOR_CPP